client/signup/difm/components/BrowserView.tsx

```diff
@@ -0,0 +1,124 @@
+import styled from '@emotion/styled';
+import {
+	ABOUT_PAGE,
+	BLOG_PAGE,
+	CONTACT_PAGE,
+	HOME_PAGE,
+	PHOTO_GALLERY_PAGE,
+	SERVICE_SHOWCASE_PAGE,
+	SITEMAP_PAGE,
+} from 'calypso/signup/difm/constants';
+import aboutPage from 'calypso/signup/difm/images/about-page.svg';
+import blogPage from 'calypso/signup/difm/images/blog-page.svg';
+import contactPage from 'calypso/signup/difm/images/contact-page.svg';
+import homePage from 'calypso/signup/difm/images/home-page.svg';
+import photoGallery from 'calypso/signup/difm/images/photo-gallery.svg';
+import serviceShowcase from 'calypso/signup/difm/images/service-showcase.svg';
+
+const Container = styled.div< { isSelected?: boolean; isClickDisabled?: boolean } >`
+	border: 2px solid
+		${ ( { isSelected } ) => ( isSelected ? 'var( --studio-blue-50 )' : '#ffffff00' ) };
```

Comment:
`#ffffff00` seems to be a typo. In the design, the border is `border: 1px solid #C3C4C7`

Reply:
Actually this is a filler border but yes I need to probably just use studio white here.

However I will be doing a change in the actual border below
https://github.com/Automattic/wp-
calypso/blob/6b323d63a60fed55cf2ea99eeafd249ae4a72c5e/client/signup/difm/components/BrowserView.tsx#L52